app/src/main/java/com/example/lhi06/residue/ReviewAdapter.java

```diff
@@ -59,10 +82,11 @@ public int getItemCount() {
 
     class ReviewViewHolder extends RecyclerView.ViewHolder {
 
-        @BindView(R.id.linearLayout_row) View rootView;
+        @BindView(R.id.review_preview_row) LinearLayout rootView;
```

Comment:
Should be as generic as possible. So if you don't need anything specific from the LinearLayout the field should be View